test/runner.js

```diff
@@ -1,56 +1,66 @@
-const config = require('./../lib/config');
-const benchpress = require('@angular/benchpress');
-const fs = require('fs-extra');
-const path = require('path');
+const config = require("./../lib/config");
+const fs = require("fs-extra");
+const path = require("path");
```

Comment:
```suggestion
```
These modules were not used within the file before the changes and after as well, so I'd propose to remove them.